docker/registry/slim-base/gradle.properties

```diff
@@ -1,3 +1,3 @@
 io.deephaven.project.ProjectType=DOCKER_REGISTRY
 deephaven.registry.imageName=ghcr.io/deephaven/server-slim-base:edge
-deephaven.registry.imageId=ghcr.io/deephaven/server-slim-base@sha256:ca9b55d2e075dc12e220b8cb13fdd9cd4cb7fc8f68dfdaba99fc240f5a916266
+deephaven.registry.imageId=ghcr.io/deephaven/server-slim-base@sha256:8011b73d42aebf3d240c25e2b0a12c21dbc0049fa8444924fe9f9dbeb1ca9ddfs
```

Comment:
This is not a valid digest. I think you added an extra "s" onto the end. Clearly, our CI build doesn't even use this image, otherwise it would have failed... :/